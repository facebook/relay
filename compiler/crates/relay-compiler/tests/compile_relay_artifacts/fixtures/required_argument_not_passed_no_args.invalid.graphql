# expected-to-throw

query requiredArgumentNotPassedNoArgsQuery {
  me {
    ...requiredArgumentNotPassedNoArgsFragment
  }
}

fragment requiredArgumentNotPassedNoArgsFragment on User @argumentDefinitions(
  sizearg: { type: "[Int!]!" }
) {
  profilePicture(size: $sizearg) {
    uri
  }
}
