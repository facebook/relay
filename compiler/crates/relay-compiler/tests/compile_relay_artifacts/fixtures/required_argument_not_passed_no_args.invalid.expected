==================================== INPUT ====================================
# expected-to-throw

query requiredArgumentNotPassedNoArgsQuery {
  me {
    ...requiredArgumentNotPassedNoArgsFragment
  }
}

fragment requiredArgumentNotPassedNoArgsFragment on User @argumentDefinitions(
  sizearg: { type: "[Int!]!" }
) {
  profilePicture(size: $sizearg) {
    uri
  }
}
==================================== ERROR ====================================
✖︎ Missing required argument `sizearg` on this fragment spread.

  required_argument_not_passed_no_args.invalid.graphql:5:5
    4 │   me {
    5 │     ...requiredArgumentNotPassedNoArgsFragment
      │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    6 │   }

  ℹ︎ defined on the fragment here

  required_argument_not_passed_no_args.invalid.graphql:10:3
    9 │ fragment requiredArgumentNotPassedNoArgsFragment on User @argumentDefinitions(
   10 │   sizearg: { type: "[Int!]!" }
      │   ^^^^^^^
   11 │ ) {
